packages/calypso-e2e/src/lib/flows/difm-flow.ts

```diff
@@ -0,0 +1,192 @@
+import { Page } from 'playwright';
+import { DataHelper } from '../..';
+
+const selectors = {
+	// General
+	button: ( text: string ) => `button:text("${ text }")`,
+	header: ( text: string ) => `h1.formatted-header__title:has-text("${ text }")`,
+
+	// Options page
+	siteNameInput: '#siteTitle',
+	taglineInput: '#tagline',
+
+	// Social page
+	twitterInput: 'input[name=twitterUrl]',
+
+	// Design page
+	freeDesignButton: 'button[data-e2e-button=freeOption]',
+
+	//Page picker page
+	pageOption: ( text: string ) => `div.css-1wedqbn:has-text("${ text }")`,
```

Comment:
Hrm, so I think that class name `.css-1wedqbn` is dynamic and programmatically generated. That is to stay, I'm not sure how reliable and stable it will stay over updates and new builds.

I took a look at the DOM for that page, and oof, we do not have a lot of good options in the existing markup! 😱 Got any recommendations for how to proceed with any area like that?